src/javascript/app_2/Utils/Validator/declarative_validation_rules.js

```diff
@@ -0,0 +1,111 @@
+import Client                    from '../../../_common/base/client_base';
+import {
+    addComma,
+    getDecimalPlaces }           from '../../../_common/base/currency_base';
+import Password                  from '../../../_common/check_password';
+import { localize }              from '../../../_common/localize';
+import { compareBigUnsignedInt } from '../../../_common/string_util';
+import { cloneObject }           from '../../../_common/utility';
+
+// ------------------------------
+// ----- Validation Methods -----
+// ------------------------------
+const validRequired     = (value/* , options, field */) => {
```

Comment:
extra whitespace before `=`

Reply:
same for the two functions below, unless for some reason we want to match them to the other functions?

Reply:
@negarn 
yes, there is extra whitespaces, but I just follow the previous rules in binary static since these snippets have been copied from the previous validation file in binary-static.